liblumen_alloc/src/erts/term/arch/tag.rs

```diff
@@ -0,0 +1,77 @@
+use core::fmt;
+
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub enum Tag<T> 
+where
+    T: Clone + Copy + PartialEq + Eq,
+{
+    #[allow(unused)]
```

Comment:
Yes, kinda - they are unused because we never have a need to construct them, but for completeness sake I think it makes sense to leave them in place, especially since the need for constructing them may change in the future.